grpc/weaviate.proto

```diff
@@ -158,6 +159,19 @@ message HybridSearchParams {
   FusionType fusion_type = 5;
 }
 
+message NearTextSearchParams{
```

Comment:
Maybe we could split this big proto file into smaller ones? I'm thinking for example about separate proto file for `filters` only `filters.proto`, we can do this later, but wanted to know your opinion on this

Reply:
yes, good idea! Not in this PR, but later